react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+export default class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      headerClick: '',
+      doubleClicked: false
```

Comment:
You can simplify this a bit. Try tracking only one thing `openHeader`, initially `null`. If the user clicks a header, set the `openHeader`. If they're clicking the same one again, set it back to `null`.